src/components/RestaurantPreview.tsx

```diff
@@ -0,0 +1,31 @@
+import "@buildo/bento-design-system/index.css";
+import "@buildo/bento-design-system/defaultTheme.css";
+import { Card, Stack, Title, Body, Label } from "@buildo/bento-design-system";
+import { useTranslation } from "react-i18next";
+
+export interface PreviewProps {
+  name: string;
+  image_url: string;
```

Comment:
Standard case style for JS/TS is camel case
```suggestion
  imageUrl: string;
```